### ISSUE TITLE
Words starting with ad or ads

### ISSUE THREAD

**Issue by Pierre-aka-Terdef** [gorhill/uBlock-for-firefox-legacy#331]
Hello,
I am the developer of computer security, privacy and web neutrality website assiste.com (https://assiste.com).
I have an old WEB page, which I hadn't looked at for a while, concerning the security tool of Merijn Bellekom (currently at Malwarebytes): Ads Spy at https://assiste.com/Logitheque/ADS_Spy.html .
This tool identifies invisible ADS (Alternate Data Stream) files.
In this page, there is an image called ADS_Spy.png and its thumbnail ADS_Spy_V.png.
I work under Firefox (currently version 100) and I installed uBlock Origin.
Unable to get those images displayed. I have to deactivate uBlock Origin to check that they do are on the server, under the good names, and that my URLs are correct.
I already had this problem with the ad-remover page (https://assiste.com/Logitheque/Ad-Remover.html) and I had to change the name of the illustration and its thumbnail.
I think uBlock Origin filters out AD or ADS prefixes systematically.
Thanks to look at that.
Pierre Pinard.

### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [ ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

[Description of the bug or feature]

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

### Expected behavior:

[What you expected to happen]

### Actual behavior:

[What actually happened]

### Your environment

* uBlock Origin version:
* Browser Name and version:
* Operating System and version:

**Comment by AroKol78** [comment]
First, this add-on doesn't support Firefox100 (max 56).
Secondly, this is not the place to report this problem.

I recommend reporting the described problem in
https://github.com/uBlockOrigin/uAssets/issues or https://github.com/easylist/easylist/issues

Have a nice day.

**Comment by Pierre-aka-Terdef** [comment]
Hi!, AroKol78
Thanks, but uBlock Origin (actualy 1.42.24) is supported by Firefox since 25.04.2005 and not before that date.
The first release for firefox was announced on April 25, 2015, i.e. version 37.0.2 released on April 20, 2015.
To date, version 1.42.4 is in the panel of Firefox extensions which has just been upgraded to version 100.0.1.
I'll post on the URL you gave.
Thanks,
Pierre Pinard
https://assiste.com

**Comment by ghost** [comment]
This is wrong repository, to report filtering issues go here: https://github.com/uBlockOrigin/uAssets

**Comment by Vangelis66** [comment]
@Pierre-aka-Terdef 

Besides this not being the correct issue tracker to report **filtering issues**, take a closer look at the name of this repo: "uBlock-for-firefox-**legacy**" 😉 
This is a **special version** of **uBO** (latest release being `1.16.4.30`) in the "legacy" **XUL format**, that **Mozilla** deprecated starting with **Firefox Quantum v57.0**; this special version of **uBO** can't be installed in **Fx 100.x.x**, because that browser only accepts add-ons of the **WebExtensions** format... 
So, you're not even using yourself the version of **uBO** maintained in this "special" repo... 

If you're curious, this "**legacy**" version of **uBO** is targeting **Firefox forks** like the **UXP** browsers (e.g. **Pale Moon**, **Basilisk** and their various forks), **WaterFox Classic** and **SeaMonkey**: 
https://github.com/gorhill/uBlock-for-firefox-legacy/tree/master/dist#install

I hope I've cleared things up for you... 😄 
This issue should be closed as `invalid`...

**Comment by krystian3w** [comment]
Maybe possible use this to run uBo Legacy in Firefox 57+:

https://github.com/xiaoxiaoflood/firefox-scripts/tree/master/extensions

**Comment by garoto** [comment]
> Maybe possible use this to run uBo Legacy in Firefox 57+

What's the point? I must be missing something.

**Comment by krystian3w** [comment]
Some people don't found alternatives in WebExtension, so made mechanism to run oldie extension with little rewrite own code if not run out of the box.

**Comment by FrostKnight** [comment]
The OP, seems confused...

Or he is trying to advertise, or he is trolling...

I dunno, all three are possible...

I have to admit though, it looks like the 2nd is more likely on closer inspection...

**Comment by krystian3w** [comment]
> trolling

He renamed file after mapx- proposition so EasyList no added exception for page:

`@@/ADS_Spy_V.png$domain=assiste.com`

**Comment by FrostKnight** [comment]
Hmm... 

So what do you think he is doing?

At this point, no idea...

Btw, I got my numbers mixed up, I should have been more clear,

I think it had to do with advertising...

Confused was option 1...

**Comment by krystian3w** [comment]
He tries fix blocked photo without rename, but recommended was rename.

But maybe didn't know what block photo (EasyList), so simplfied issue to no possible reproduce with ABP/AdBlock/AdGuard/Prioxy.

**Comment by FrostKnight** [comment]
> He tries fix blocked photo without rename, but recommended was rename.
> 
> But maybe didn't know what block photo (EasyList), so simplfied issue to no possible reproduce with ABP/AdBlock/AdGuard/Prioxy.

Well regardless, it is just plain weird then...
Hopefully he finds a solution to his problem...
